Author: Ron Lee <ron@debian.org>
Bugs-Debian: #336188, #363999
Origin: vendor
Forwarded: not-yet
Description: Don't fail to build the package if the tests are not
 successful. Some of them require network access for MX checking, which is
 good to check, but should not prohibit people offline from building it (and
 ignoring tests that cannot succeed in that environment). Closes: #336188
 thanks!

--- libemail-valid-perl.orig/t/valid.t
+++ libemail-valid-perl/t/valid.t
@@ -90,8 +90,11 @@
 );
 
 SKIP: {
-  skip "your dns appears missing or failing to resolve", 2
-    unless eval { $v->address(-address=> 'devnull@pobox.com', -mxcheck => 1) };
+  skip "Net::DNS not available or network down", 2
+    unless eval { require Net::DNS; 1; } && Net::DNS::mx('debian.org');
+
+#  skip "your dns appears missing or failing to resolve", 2
+#    unless eval { $v->address(-address=> 'devnull@pobox.com', -mxcheck => 1) };
 
   if (
     $v->address(-address => 'blort@will-never-exist.pobox.com', -mxcheck => 1)
